driver/parser/parser_test.go

```diff
@@ -168,9 +168,7 @@ nodes:
     start: 5
     end: 7.5
     genesis:
-      import:
-        start: 3
-        path: /path/to/genesis
+      import: /path/to/genesis
```

Comment:
This one is mine - but would appreciate your help looking into this.
Looks to me like this should have failed since genesis import looks for `.g` extension but the test passed historically.

Reply:
Well in this file, we test only syntactic errors with `TestParseWithGenesisImportWorks`. Semantic errors are not tested here. Or am I wrong?

Reply:
You're absolutely right, this is simply testing the parser.  Please ignore.